src/picasso-definition/components/navigation-panel/index.js

```diff
@@ -0,0 +1,101 @@
+import NUMBERS from '../../../constants/numbers';
+import KEYS from '../../../constants/keys';
+
+// Translation (up/down/left/right)
+function translate({ viewHandler, direction, percent }) {
+  const { xAxisMax, xAxisMin, yAxisMax, yAxisMin } = viewHandler.getDataView();
+  let deltaX = 0;
+  let deltaY = 0;
+  if (direction === 'x') {
+    deltaX = ((xAxisMax - xAxisMin) * percent) / 100;
+  } else {
+    deltaY = ((yAxisMax - yAxisMin) * percent) / 100;
+  }
+  const dataView = {
+    xAxisMin: xAxisMin + deltaX,
+    xAxisMax: xAxisMax + deltaX,
+    yAxisMax: yAxisMax + deltaY,
+    yAxisMin: yAxisMin + deltaY,
+  };
+  viewHandler.setDataView(dataView);
+}
+
+// Zoom in/ Zoom out
```

Comment:
Sometimes comments are warrant, but if a description like this is needed then the function is named incorrectly. Try to remove the comments and name the function to something better describing what it does instead.

Same for comments above for `translate` function.